eessi_container.sh

```diff
@@ -173,6 +173,11 @@ while [[ $# -gt 0 ]]; do
       export https_proxy=${HTTPS_PROXY}
       shift 2
       ;;
+    --)
```

Comment:
We should make sure the use of `--` is also reflected in the docs at https://eessi.github.io/docs/getting_access/eessi_container/#running-a-simple-command

Reply:
See [PR#103](https://github.com/EESSI/docs/pull/103)